pauthabielf64/pauthabielf64.rst

```diff
@@ -206,6 +207,14 @@ This document is at **Alpha** release quality.
   |            |                     | descriptions of existing encoding and relocations. Add           |
   |            |                     | description of SHT_AUTH_RELR.                                    |
   +------------+---------------------+------------------------------------------------------------------+
+  | 0.3        | 22nd October 2020   | Delete the appendix giving an encoding for GOT generating        |
+  |            |                     | relocations using the relocation addend to carry the signing     |
+  |            |                     | signing schema.                                                  |
+  |            |                     | Move the remaining GOT signing text to an optional appendix      |
+  |            |                     | Describe the default dlsym signing schema and add .symauth and   |
+  |            |                     | .dynauth as an appendix                                          |
+  |            |                     | Provide details for the ELF marking scheme                       |
+  +------------+---------------------+------------------------------------------------------------------+
```

Comment:
There needs to be at least one line of separation between the different points, otherwise they will be written on the same line, as they now do (click `view file` from the drop-down list). I personally prefer for them to have a period at the end, or for the sentence not to be capitalized.

Reply:
I've rearranged the table to finish with a full-stop and to add one line of separation.